runtime/catalog-karapace/src/main/java/io/aklivity/zilla/runtime/catalog/karapace/internal/KarapaceCatalogHandler.java

```diff
@@ -190,22 +344,18 @@ private String sendHttpRequest(
                 .build();
         // TODO: introduce interrupt/timeout for request to schema registry
 
+        String responseBody;
         try
         {
             HttpResponse<String> httpResponse = client.send(httpRequest, HttpResponse.BodyHandlers.ofString());
             boolean success = httpResponse.statusCode() == 200;
-            String responseBody = success ? httpResponse.body() : null;
-            if (!success)
-            {
-                event.remoteAccessRejected(catalogId, httpRequest, httpResponse.statusCode());
-            }
-            return responseBody;
+            responseBody = success ? httpResponse.body() : null;
```

Comment:
We don't really need `success` any more, right?
```suggestionif (
responseBody = httpResponse.statusCode() == 200 ? httpResponse.body() : null;
```